app/src/main/java/fr/neamar/kiss/normalizer/DateNormalizer.java

```diff
@@ -0,0 +1,13 @@
+package fr.neamar.kiss.normalizer;
+
+import android.text.format.DateFormat;
+
+public class DateNormalizer {
+    public static String normalizeDate(Long lastTimeContacted) {
+        if(lastTimeContacted == null || lastTimeContacted == 0) {
+            return "";
+        }
+
+        return DateFormat.format("HH:mm MM/dd/yy", lastTimeContacted).toString();
```

Comment:
I agree with @Pluggi, it's better if we can use the user's locale instead than a hardcoded format